### DISCUSSION TITLE
404 ERROR ON VERCEL-PREVIEW --> SOLVED ISSUE FOR ANYONE WHO GETS THE SAME ERROR:

### DISCUSSION THREAD

**Discussion by StefanFitoussi** [neuefische/questions#322] [Web] [answered]
SOLVED ISSUE FOR ANYONE WHO GETS THE SAME ERROR:

Vercel was connected to an empty repository. A NEXT-repository then was cloned into the first one. Vercel did not realise that it now was a NEXT-repository and threw a 404-error in preview and the messages in the vercel-building-tab visible on the screenshots below.

You can fix this by going to vercel and to your projects-page. Choose the project in question and switch to the respective repository-mode (in this case: NEXT) in the dropdown-menue. Then redeploy the project and it should work :)

![Bildschirmfoto 2022-11-17 um 16 31 05](https://user-images.githubusercontent.com/113462689/202500094-8ae30c2a-64e0-471f-b55a-a50583b2ace4.png)
![Bildschirmfoto 2022-11-17 um 16 31 21](https://user-images.githubusercontent.com/113462689/202500099-5a63331d-d97e-4c9e-b8d8-5ac81bdd1418.png)
![Bildschirmfoto 2022-11-17 um 16 31 32](https://user-images.githubusercontent.com/113462689/202500108-bb1b6f87-5068-46c8-9ef5-9b4bb142d691.png)
![Bildschirmfoto 2022-11-17 um 16 32 03](https://user-images.githubusercontent.com/113462689/202500111-2fee79b4-5694-47bb-bdf0-75acd2a47b7e.png)
![Bildschirmfoto 2022-11-17 um 16 32 15](https://user-images.githubusercontent.com/113462689/202500114-ffdf0628-668b-4109-a854-d0b48361cb61.png)

**Comment by ahohnsen** [comment, marked as answer]
Go to your project on Vercel -> `Settings` -> `Build & Development Settings` -> Choose `Framework Preset` from dropdown

Thank you for posting @StefanFitoussi  and thanks to @Kheirah for providing the solution :)

**Reply by StefanFitoussi** [reply]
Thanks to everyone for having helped solve this issue :)
